src/components/CodeHighlight/LogColorizer.css

```
@@ -0,0 +1,69 @@
+.toolbar {
+  position:sticky;
```

```suggestion
  position: sticky;
```
Please handle other places for consistency.

---

Done